src/Identifier/VerificationIdentifier.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * CakePHP(tm) : Rapid Development Framework (http://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ * @link          http://cakephp.org CakePHP(tm) Project
+ * @license       http://www.opensource.org/licenses/mit-license.php MIT License
+ */
+namespace Authentication\Identifier;
+
+use Authentication\Identifier\Resolver\ResolverAwareTrait;
+
+/**
+ * Identity verification identifier
+ */
+class VerificationIdentifier extends AbstractIdentifier
```

Comment:
How would this work in practice with other identifiers? If I made an identifier collection with this and PasswordIdentifier, couldn't I become any user by 'logging in' with just an id?

Reply:
Only authenticators have access to identifiers so if an authenticator pass the user provided id with the `id` data key to the identifiers it could log in as any user. But currently there’s no such authenticator. FormAuthenticator uses `username/password` data keys and Token operate on `token` key.

Reply:
Ok. But right now FormAuthenticator passes the entire request body into the identifiers. https://github.com/cakephp/authentication/blob/0ecd87e1cdd22719c30c43b3327b63814de8e3dc/src/Authenticator/FormAuthenticator.php#L82-L83

Which is why I'm concerned.

Reply:
You are right, this should be fixed.

Reply:
PR is open #153